src/fread.c

```diff
@@ -2384,8 +2396,10 @@ int freadMain(freadMainArgs _args)
         stopTeam = true;
       }
       prepareThreadContext(&ctx);
-    
-      #pragma omp for ordered schedule(dynamic) reduction(+:thRead,thPush) reduction(max:max_col)
+      
+      double th_read_openmp = timestamps.th_read;//pragma workaround
+      double th_push_openmp = timestamps.th_push;
+      #pragma omp for ordered schedule(dynamic) reduction(+:th_read_openmp,th_push_openmp) reduction(max:max_col)
```

Comment:
> For some reason it won't compile (relating to an openmp pragma).
>
> If anyone knows what this is about, I would love to be enlightened.

The reduction clause here is being applied to local variables that are private to each thread, not shared variables.